docs/deployment/gunicorn.rst

```diff
@@ -35,16 +35,14 @@ accesslog lines to stderr and use the debug loglevel.
     bind = unix:/var/run/app.sock
     workers = 4
     preload = true
-    reload = true
     accesslog = -
     loglevel = debug
 
 For all configuration options that may be used, have a look at the `available
 settings <http://docs.gunicorn.org/en/stable/settings.html>`_.
 
 Keep in mind that settings defined within a gunicorn configuration file or
-command line arguments given to gunicorn take precedence over the settings
-established within the INI file.
+take precedence over the settings established within the INI file.
```

Comment:
This sentence does not make sense, maybe remove the "or" from line 44?

Reply:
Thank you!